### DISCUSSION TITLE
Unable to get payments or orders associated with a subscription

### DISCUSSION THREAD

**Discussion by gsumo** [swellstores/community#194] [Help] [answered]
I am attempting to show the payment history for a subscription to a logged in user.

I have attempted to do it this way:
const { results, page_count, page } = await swell.subscriptions.list({
  expand: ["payments"],
});

and gotten an empty list

<img width="266" alt="Screenshot 2023-04-03 at 5 29 53 PM" src="https://user-images.githubusercontent.com/106490529/229655708-f0a6e21a-3a3f-4987-b559-fcde6033e74b.png">

And with graphql I had the same problem, payments being an invalid field.

**Comment by sags95** [comment]
@gsumo What is the status of these payments in Swell admin?

**Reply by gsumo** [reply, marked as answer]
const result = await swell.subscriptions.list({
  expand: ["orders"],
});
This was the solution, thanks
